docker/Dockerfile

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+ARG ENV="cpu"
+ARG HOME="/root"
+
+FROM ubuntu:18.04 AS cpu
+
+ARG HOME
+ENV HOME="${HOME}"
+WORKDIR ${HOME}
+
+# Install base dependencies
+RUN apt-get update && \
+    apt-get install -y curl git build-essential
+
+# Install Anaconda
+ARG ANACONDA="https://repo.continuum.io/miniconda/Miniconda3-4.6.14-Linux-x86_64.sh"
+RUN curl ${ANACONDA} -o anaconda.sh && \
+    /bin/bash anaconda.sh -b -p conda && \
+    rm anaconda.sh 
+ENV PATH="${HOME}/conda/bin:${PATH}"
+ENV PATH="${HOME}/conda/envs/cv/bin:${PATH}"
+
+# Clone Computer Vision repo
+ARG BRANCH="master"
+RUN git clone --depth 1 --single-branch -b ${BRANCH} https://github.com/microsoft/computervision
+
+# Setup Jupyter notebook configuration
+ENV NOTEBOOK_CONFIG="${HOME}/.jupyter/jupyter_notebook_config.py"
+RUN mkdir ${HOME}/.jupyter && \
+    echo "c.NotebookApp.token = ''" >> ${NOTEBOOK_CONFIG} && \
+    echo "c.NotebookApp.ip = '0.0.0.0'" >> ${NOTEBOOK_CONFIG} && \
+    echo "c.NotebookApp.allow_root = True" >> ${NOTEBOOK_CONFIG} && \
+    echo "c.NotebookApp.open_browser = False" >> ${NOTEBOOK_CONFIG} && \
+    echo "c.MultiKernelManager.default_kernel_name = 'python3'" >> ${NOTEBOOK_CONFIG}
+
+
+# GPU Stage
+FROM nvidia/cuda:9.0-base AS gpu
+
+ARG HOME
+WORKDIR ${HOME}
+
+COPY --from=cpu ${HOME} .
+
+ENV PATH="${HOME}/conda/bin:${PATH}"
+ENV PATH="${HOME}/conda/envs/cv/bin:${PATH}"
+
+
+# Final Stage
+FROM $ENV AS final
+
+# Install Conda dependencies
+RUN conda env update -f computervision/environment.yml && \
```

Comment:
This updates the base env, so there won't be a CV env created. You should remove the previous lines adding the CV env to the path, or create a new env and remove the base env paths

Reply:
Slightly different from reco, the yaml file that CV is using has the name specified so the `conda env` creates a cv env by default.

Reply:
But this is running conda update. That will still create a new env?

Reply:
Ah I see if not exist `conda update` will create a new one but yea it is more clear to use `conda create` in this context. 

Updated the PR.

Reply:
Ok, I didn't realize that, but I agree making it more explicit is good. In this case the two lines that add the base probably should be removed

```ENV PATH="${HOME}/conda/bin:${PATH}"```

this might lead to confusion if something breaks in the cv env it would fall back to the base which wouldn't have the right packages.

Reply:
Yea we can remove this.